lib/spl-build.js

```diff
@@ -346,48 +346,21 @@ export default {
             label: 'IBM Streams',
             shouldDisplay: self.shouldShowMenu,
             beforeGroupContaining: ['tree-view:open-selected-entry-up'],
-            submenu: [
-              {
-                label: 'Build',
-                command: 'spl-build:build-download',
-                shouldDisplay: self.shouldShowMenuSpl
-              },
-              {
-                label: 'Build and submit job',
-                command: 'spl-build:build-submit',
-                shouldDisplay: self.shouldShowMenuSpl
-              },
-              {
-                label: 'Build',
-                command: 'spl-build:build-make-download',
-                shouldDisplay: self.shouldShowMenuMake
-              },
-              {
-                label: 'Build and submit job(s)',
-                command: 'spl-build:build-make-submit',
-                shouldDisplay: self.shouldShowMenuMake
-              },
-              {
-                label: 'Submit job',
-                command: 'spl-build:submit',
-                shouldDisplay: self.shouldShowMenuSubmit
-              },
-              {
-                label: 'Open IBM Cloud Private for Data dashboard',
-                command: 'spl-build:open-icp4d-dashboard',
-                shouldDisplay: self.shouldShowMenuV5.bind(self)
-              },
-              {
-                label: 'Open IBM Cloud dashboard',
-                command: 'spl-build:open-IBM-cloud-dashboard',
-                shouldDisplay: self.shouldShowMenuV4.bind(self)
-              },
-              {
-                label: 'Open IBM Streams Console',
-                command: 'spl-build:open-streams-console',
-                shouldDisplay: self.shouldShowMenu.bind(self)
-              }
-            ]
+            submenu: self.menu(self)
+          },
+          {
+            type: 'separator'
+          }
+        ],
+        'atom-text-editor': [
+          {
+            type: 'separator'
+          },
+          {
+            label: 'IBM Streams',
+            shouldDisplay: self.shouldShowMenu,
+            beforeGroupContaining: ['tree-view:open-selected-entry-up'],
```

Comment:
```suggestion
            beforeGroupContaining: ['core:undo'],
```

To make the streams submenu show up first, need to set this to one of the commands that shows in the editor context menu.